setup.py

```diff
@@ -33,9 +33,10 @@
         'fsspec',
         'aiohttp',
     ],
-    entry_points={
-        "xarray.backends": ["xsar=xsar.xarray_backends:XsarXarrayBackend"]
-    },
+    # disabled while migrating from rasterio to rioxarray
+    #entry_points={
+    #    "xarray.backends": ["xsar=xsar.xarray_backends:XsarXarrayBackend"]
+    #},
```

Comment:
Removing comment line

Reply:
migration to rioxarray is done, i'm going to try to reactivate this backend